pkg/controller/node/node_controller.go

```diff
@@ -285,9 +284,6 @@ func NewNodeController(
 		serviceCIDR:            serviceCIDR,
 		allocateNodeCIDRs:      allocateNodeCIDRs,
 		allocatorType:          allocatorType,
-		forcefullyDeletePod: func(p *v1.Pod) error {
-			return util.ForcefullyDeletePod(kubeClient, p)
```

Comment:
You should also delete function definition. I think it's used only here.

Reply:
indeed missed this one